src/main/webapp/app/shared/util/firebase/firebase-review-utils.tsx

```diff
@@ -230,41 +200,56 @@ export const reformatReviewTitle = (baseReviewLevel: BaseReviewLevel) => {
   );
 };
 
-export const reviewLevelSortMethod = (a: ReviewLevel, b: ReviewLevel) => {
-  const reviewActionPriority = [ReviewAction.NAME_CHANGE, ReviewAction.UPDATE, ReviewAction.CREATE, ReviewAction.DELETE];
-  const aIndex = reviewActionPriority.indexOf(a.reviewAction);
-  const bIndex = reviewActionPriority.indexOf(b.reviewAction);
-  if (aIndex === -1) {
-    return 1;
-  }
-  if (bIndex === 1) {
-    return -1;
+export const reviewLevelSortMethod = (a: BaseReviewLevel, b: BaseReviewLevel) => {
```

Comment:
Are you doing this case so that it can determine whether its a `BaseReviewLevel` or not by checking for `reviewInfo`? If so, would you consider checking the types early and doing an early return? It is slightly more verbose, but to me it is more understandable.  This is preference though, so if you disagree, feel free to disregard

Reply:
Good point, I will try to make it explicit. 
I want to sort the `MetaReviewLevel` between Updates and Creates, but that type does not have `reviewInfo`